### ISSUE TITLE
Request: GRUB config consolidation

### ISSUE THREAD

**Issue by lowland3r** [PartialVolume/shredos.x86_64#286]
I'm not too familiar with the inner workings of GRUB, but for the sake of simplicity would it be possible for both the BIOS and EFI loader to use the same GRUB config file?

**Comment by PartialVolume** [comment]
> I'm not too familiar with the inner workings of GRUB, but for the sake of simplicity would it be possible for both the BIOS and EFI loader to use the same GRUB config file?

Yes, most probably the same file could be used. Historically there was always two and I vaguely remember a reason why that was an advantage in some cases, but for the life of me, I don't remember now.

I would think changing that now is low priority as there are other more important things to be done. It's a shame FAT doesn't support symbolic links as that would be a easy fix anybody could do.
